flink-tests/src/test/java/org/apache/flink/runtime/jobmaster/JobMasterStopWithSavepointITCase.java

```diff
@@ -99,31 +94,6 @@
 
     private JobGraph jobGraph;
 
-    @Test
-    public void suspendWithSavepointWithoutComplicationsShouldSucceedAndLeadJobToFinished()
```

Comment:
This test reimplements partial logic of `StreamTask` in `NoOpBlockingStreamTask`. At the same time, we cover a successful savepoint taking e.g. in `StreamTaskFinalCheckpointsTest`